### DISCUSSION TITLE
"BOINC Manager is not able to start a BOINC client" Daemon Start Failed

### DISCUSSION THREAD

**Discussion by phroggy07** [BOINC/boinc#5758] [Q&A] [answered]
BOINC has been running easily for years, then all of a sudden, I get the below when I restart my computer, and when I try to restart the BOINC manager. The manager starts, but is blank and won't let any projects run.

![Screenshot 2024-08-12 122038](https://github.com/user-attachments/assets/26929a05-f1ae-47ab-ab2c-95f875eb1baa)

I tried following the steps noted, but the BOINC client doesn't appear in the "Services" applet.

I am using an administrator account, and this is on a Windows 11 laptop. I even tried reinstalling Windows, but it may have been caused by a recent update.

**Comment by AenBleidd** [comment]
@phroggy07, could you please go to the installation folder (by default 'C:\Program Files\BOINC') and check that boinc.exe is located in that folder?

**Comment by phroggy07** [comment]
There are three applications in the installation folder: boinccmd, boincmgr and boincscr, but boinc.exe is not included.

**Comment by AenBleidd** [comment]
@phroggy07, ok, looks like some end-point protection (antivirus) software removed your boinc.exe file.
- What version of BOINC have you installed?
- What is the antivirus software you have installed?
- Could you please try to add BOINC installation folder to the exceptions in that antivirus software?

**Reply by phroggy07** [reply]
Thanks. It ended up being Webtoot, which decided bounce was a threat after about five years. Got it reinstalled, boinc.exe set as trusted, and it seems to be working

**Comment by RichardHaselgrove** [comment, marked as answer]
To amplify that last point about antivirus software: it's normally the 'heuristic' (behavioural) component of modern AV software which causes this problem: the scanning component doesn't see anything wrong, so disabling it won't help.

Download and install a replacement copy of boinc.exe, but don't let it run just yet. Find boinc.exe in the program folder as before, and right-click on it: look for any entries relating to your AV supplier. Explore any options that don't refer to scanning, and see if it has any options for disabling that particular function for this file.

For example: in Norton 360, the relevant component is called 'File Insight', and it allows you to 'Trust this file manually'.

You may also have to do this for new versions of any science applications issued by any of the projects you run.

**Reply by phroggy07** [reply]
Yep, it was Webroot. Got it fixed. Thanks for the insight

**Reply by AenBleidd** [reply]
Yep, looks like a known false positive detection from them: https://www.virustotal.com/gui/file/91e405e8a527023fb8696624e70498ae83660fe6757cef4871ce9bcc659264d3

**Comment by AenBleidd** [comment]
Ok, since this is clearly not the issue on the BOINC side, I'm transferring it to discussion.
